pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -1452,9 +1484,9 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
           await this.progressTracker.action('Starting Kubernetes', 100, this.kubeBackend.start(config, kubernetesVersion));
         }
 
-        // If it's not a privileged installation preset guestAgent
-        // port binding address to localhost only.
-        if (!this.privilegedServiceEnabled && !config.kubernetes.ingress.localhostOnly) {
+        // Set the kubernetes ingress address to localhost only for
+        // a non-admin installation, if it's not already set.
+        if (!await this.getIsAdminInstall() && !config.kubernetes.ingress.localhostOnly) {
```

Comment:
This would be _slightly_ faster if we reverse the conditions (because reading the config doesn't need `await`).
But that's not going to block merging the PR.